lldb/source/Plugins/LanguageRuntime/ObjC/AppleObjCRuntime/AppleObjCClassDescriptorV2.cpp

```diff
@@ -668,6 +668,18 @@ uint64_t ClassDescriptorV2::GetInstanceSize() {
   return 0;
 }
 
+// From the ObjC runtime.
+static uint8_t IS_SWIFT_STABLE = 1U << 1;
+
+bool ClassDescriptorV2::IsSwift() const {
```

Comment:
Can this be 

LanguageType ClassDescriptorV2::GetImplementationLanguage()

and similar in the other places where you use IsSwift?  That would make uses like:

```
      if (descriptor_sp->IsSwift())
        // This class is implemented in Swift, making it the preferred runtime.
        if (auto *swift_runtime =
                process_sp->GetLanguageRuntime(lldb::eLanguageTypeSwift))
          return swift_runtime;
    }

```

more natural, like:
      LanguageType imp_lang = descriptor_sp->GetImplementationLanguage()
      if (imp_lang != eLanguageTypeUnknown)
        if (auto *imp_runtime =
                process_sp->GetLanguageRuntime(imp_language))
          return imp_runtime;
  }

Reply:
thanks, I like it

Reply:
changes made here [D152837](https://reviews.llvm.org/D152837). I will rebase this to include that.